source/Cosmos.System2/Network/ARP/ARPPacket.cs

```diff
@@ -1,17 +1,44 @@
-﻿using Cosmos.HAL;
+﻿using sys = System;
```

Comment:
Not sure about if this addition can break anything.
would love to hear your thoughts.